example-package-no-tests/elm.json

```diff
@@ -5,7 +5,7 @@
     "license": "BSD-3-Clause",
     "version": "1.0.0",
     "exposed-modules": [
-        "Example"
+        "Something"
```

Comment:
This example is supposed to fail because of no tests, not because of wrong `exposed-modules`.

Reply:
Thanks for catching!

Reply:
Merged in https://github.com/rtfeldman/node-test-runner/pull/448